### ISSUE TITLE
(Re)Introduce the `--version` flag on the Rust binary

### ISSUE THREAD

**Issue by Kerollmops** [meilisearch/meilisearch#2315]
I am not sure how we lost that, where the `--version`/`-v` flag goes but I am sure we had one. We must reintroduce it.

I downloaded the binary by using [the quick start guide and cURL](https://docs.meilisearch.com/learn/getting_started/quick_start.html#local-installation) and I was astonished to see that there was no way to know the version of the binary without directly running it. Even the `--help` option isn't giving us any more information. I downloaded the v0.26.1 version.

```
./meilisearch --version
error: Found argument '--version' which wasn't expected, or isn't valid in this context

	If you tried to supply `--version` as a value rather than a flag, use `-- --version`

USAGE:
    meilisearch [OPTIONS]

For more information try --help
```

```
./meilisearch -v
error: Found argument '-v' which wasn't expected, or isn't valid in this context

	If you tried to supply `-v` as a value rather than a flag, use `-- -v`

USAGE:
    meilisearch [OPTIONS]

For more information try --help
```

**Comment by MarinPostma** [comment]
This could be due to the move from structopt to clap

**Comment by curquiza** [comment]
Closed by #2316 thanks to @sanders41
